Celeste.Mod.mm/Mod/Everest/Extensions.cs

```diff
@@ -335,11 +335,30 @@ public static bool IsDown(this TouchLocationState state)
         public static bool IsUp(this TouchLocationState state)
             => state == TouchLocationState.Released || state == TouchLocationState.Invalid;
 
+        public static bool IsSafe(this Type type) {
+            try {
+                // "Probe" the type
+                _ = type.Name;
+                _ = type.Assembly.FullName;
+                _ = type.Module.FullyQualifiedName;
+
+                // Check declaring and base type
+                if (!type.DeclaringType?.IsSafe() ?? false)
+                    return false;
+                if (!type.BaseType?.IsSafe() ?? false)
```

Comment:
The inversion using `!` should have operator precedence over the null coalescing operator (see <https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#operator-precedence>), which means that the nullable is inverted first before having its default value be applied.

Reply:
> > The predefined unary and binary [operators](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/) or any overloaded operators that are supported by a value type T are also supported by the corresponding nullable value type T?. These operators, also known as lifted operators, produce null if one or both operands are null; otherwise, the operator uses the contained values of its operands to calculate the result.
> 
> I.e. `!null` is just `null`, so I don't see why that would change anything. Did you _test_ this PR btw?

Exactly - base type is `null`, which means both `type.BaseType?.IsSafe()` and `!type.BaseType?.IsSafe()` are `null`, which coalesces to `false`, meaning the if condition isn't met and the `return false;` isn't run.

Reply:
Oh I am clueless